### PR TITLE
Setting up CI

### PR DESCRIPTION
**Pull request by BeOff**
# [Link to ticket](https://trello.com/c/mTmE1ukW/64-set-up-continuous-integration)
## What it does:
sets up continuous integration with Travis.

### CODE REVIEW

**Comment by BeOff** [comment]
@alexshgov consider adding some status badges to README: Travis CI build status, test coverage percent and so on...

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: ruby
+cache: bundler
+script:
+  - rake db:setup
```
Comment (alexshgov) [inline .travis.yml]: let's move rails db setup to `before_script` block

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  - rake
```
Comment (alexshgov) [inline .travis.yml]: let's be specific in what we're running
specify commands explicitly like:
```
- rubocop
- rspec
```

File: .travis.yml (hunk 1 above)
Comment (BeOff) [reply .travis.yml]: Done!

File: .travis.yml (hunk 2 above)
Comment (BeOff) [reply .travis.yml]: Done!

**Comment by BeOff** [comment]
before_script :'-|

**Comment by labs-ruby** [comment]
* Resolve conflicts
* No need to run rubocop against config folder, exclude it via rubocop config

**Pull request merged**